server/controllers/role_controller.js

```diff
@@ -0,0 +1,31 @@
+var Role = require('../models/role');
+
+module.exports = {
+  get: function (req, res) {
+    Role.find(function (err, roles) {
+      if (err) {
+        res.status(400).send({ message: 'Error occured during request' });
+      } else {
+        res.json(roles);
+      }
+    });
+  },
+  create: function (req, res) {
+    var possibleRoles = Role.schema.path('title').enumValues;
```

Comment:
You don't need to do this. If the role is not allowed it will return an error. You can then parse the error and return the appropriate message.